Sources/Tokenizer/Layout/Constraints/Constraint.swift

```diff
@@ -38,7 +38,18 @@ public struct Constraint {
     public static func constraints(name: String, attribute: XMLAttribute) throws -> [Constraint] {
         let layoutAttributes = try LayoutAttribute.deserialize(name)
         let tokens = Lexer.tokenize(input: attribute.text)
-        return try layoutAttributes.flatMap { try ConstraintParser(tokens: tokens, layoutAttribute: $0).parse() }
+        let constraints = try layoutAttributes.flatMap { try ConstraintParser(tokens: tokens, layoutAttribute: $0).parse() }
+
+        guard
+            constraints.count > 1,
+            let field = constraints.first?.field
```

Comment:
Isn't this a potential problem? You got this field from every individual constraint before.

Reply:
I don't think so, here you can get only constraints belonging to the same `constraintField`, eg `fillHorizontally` returns two constraints, one `leading` and one `trailing`